landscape/categories.md

```diff
@@ -23,24 +23,29 @@ sub-categories. The remaining text is a description of each sub-category.
   - _Secret Management_
     - Tools that help to manage, encrypt or distribute secrets and other sensitive information.
 
-- **Identity & Access Control**
-  - _Identity_
-    - SPIFFE, identity providers, OpenID, LDAP, Okta
+- **Identity Lifecycle and & Access Management**
+  - _Identity Lifecycle_
+    - Systems for creating, governing and terminating identities and entitlements for users and systems
+      - Lifecycle management tools like Okta, FreeIPA
+      - Service identity for identifying ephemeral workloads, e.g. Spiffe
```

Comment:
If we wanted to include commercial entities, I'd point to commercial implementations like Scytale or Tremolo here as well.

Reply:
I don't think we want to have too many details of specific projects here since the document is more of a description of how to classify things... And based on that, we would come out with a landscape document that actually contains the actual implementations/projects of the ecosystem.